foursight_core/environment.py

```diff
@@ -63,12 +63,16 @@ def list_unique_environment_names(self) -> List[EnvName]:
             result.add(infer_foursight_from_env(envname=env))
         return sorted(result)  # a list and sorted
 
+    cache_list_environment_names = None
     def list_environment_names(self) -> List[EnvName]:
         """
         Lists all environments in the foursight-envs s3.
 
         Returns: a list of names
         """
+        if self.cache_list_environment_names:
+            print("foursight_core/list_environment_names: cache hit")
```

Comment:
Should this debugging statement be removed? I think dcicutils has a test for things like this.